storage/src/storage/sqlite/sync.rs

```diff
@@ -519,6 +523,68 @@ impl SyncStorage for SqliteStorage {
         Ok(out)
     }
 
+    fn get_block_digest_tree(&mut self, block_hash: &Digest) -> Result<DigestTree> {
+        let mut nodes: HashMap<Digest, Vec<Digest>> = HashMap::new();
```

Comment:
Futur perf suggestion: leverage `hash_hasher` (see #1047) since this is storing digests?